generator/views.py

```diff
@@ -214,7 +214,7 @@ def get(cls, request, share_id):
                    'is_race_seed': game.race_seed,
                    'share_info': share_info.getvalue()}
 
-        return render(request, 'generator/seed.html', context)
+        return redirect('/share/' + game.share_id)
```

Comment:
Same as above.  We can remove the share/context info since it's not being used in the render call anymore.